smartseq2_single_sample/run_multi_samples/scripts/README.md

```diff
@@ -0,0 +1,41 @@
+# Running Time and Cost Calculation
+This script is original post in [dsde-pipeline](https://github.com/broadinstitute/dsde-pipelines/blob/develop/scripts/calculate_cost.py), which is used to calculate running time and cost of workflow in google cloud. 
+We modified this python script in order to calculate cost of a subworkflow.
+# Requirement
+- Python3
+- Required modules
+ --------
+ ```python
+ import argparse
+import json
+import math
+import urllib.request, urllib.error, urllib.parse
+import base64
```

Comment:
Can you also remove `base64`, `sys` and `re` from the README.md?

Reply:
Yes. just did